src/app/organisms/room/common.jsx

```diff
@@ -173,11 +206,19 @@ function parseTimelineChange(mEvent) {
     variant,
     content,
   });
+  const type = mEvent.getType();
   const content = mEvent.getContent();
   const prevContent = mEvent.getPrevContent();
   const sender = mEvent.getSender();
   const senderName = getUsername(sender);
   const userName = getUsername(mEvent.getStateKey());
+  
+  
+  switch (type) {
+    case 'm.room.name': return makeReturnObj('edit', tJSXMsgs.roomNameChanged(senderName, content.name));
```

Comment:
There should be a seperate message for removing the room name (the room name being changed to `''`)
Maybe for topics as well.